docs/knowledge/job-market/changelog.md

```
@@ -1 +1,9 @@
 > append only
+
+## 2026-04-16
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)